gunicorn_config.py

```diff
@@ -10,15 +10,18 @@
 bind = "0.0.0.0:{}".format(os.getenv("PORT"))
 accesslog = '-'
 
-# See AWS doc
+# To avoid load balancers reporting errors on shutdown instances, see AWS doc
 # > We also recommend that you configure the idle timeout of your application
-# to be larger than the idle timeout configured for the load balancer.
+# > to be larger than the idle timeout configured for the load balancer.
 # > By default, Elastic Load Balancing sets the idle timeout value for your load balancer to 60 seconds.
 # https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#connection-idle-timeout
 in_production = os.environ.get("NOTIFY_ENVIRONMENT", "") == "production"
 if in_production:
     keepalive = 75
 
+if in_production:
+    graceful_timeout = 20
```

Comment:
Would move the new line in the same `if` statement.

Reply:
Okay I replicated the style format of the previous line of codes with the `keepalive` config. I will adjust others too then.

Reply:
The build does not pass the style check this way so I will have to revert:

```
Requirements check passed
./gunicorn_config.py:19:17: E701 multiple statements on one line (colon)
./gunicorn_config.py:30:17: E701 multiple statements on one line (colon)
```